### PR TITLE
galaxian.cpp: Identified three inputs for thepitm (MT08280)

### PR DESCRIPTION
**Pull request by scoomby**
Unusually, this game considers a single coin to be a single credit which means you could need 2 credits (coins) before a game can start depending on the Coinage DIP switch setting.  I have tried to come up with suitable descriptions for the Coinage DIP switch to avoid confusion.  Note that the Coinage DIP switch on the original thepit set is also affected which I will capture in a separate PR.  If there is a better description to use, please let me know.  Thanks in advance.

### CODE REVIEW

**Comment by Tafoid** [comment]
Please take note from the large amount of DEF_STR entries in source already which should cover most of your needs for DIP labels and identifications and shouldn't require too much use of specific strings.

https://github.com/mamedev/mame/blob/master/src/emu/ioport.cpp#L168 under "Common Shared Strings"
Most of your coinage options, for instance, are already covered.

**Comment by scoomby** [comment]
> Please take note from the large amount of DEF_STR entries in source already which should cover most of your needs for DIP labels and identifications and shouldn't require too much use of specific strings.
> 
> https://github.com/mamedev/mame/blob/master/src/emu/ioport.cpp#L168 under "Common Shared Strings" Most of your coinage options, for instance, are already covered.

Thanks but in this case, I don't think I can use the DEF_STR.  1 coin always is 1 credit (except for Free Play setting) hence I had to come up with something different.

File: src/mame/drivers/galaxian.cpp [hunk 1]
```diff
@@ -5079,38 +5079,26 @@ static INPUT_PORTS_START( thepitm )
...
-	PORT_DIPNAME( 0x40, 0x00, DEF_STR( Unknown ) ) // turning both of these on boots with 9 credits?
-	PORT_DIPSETTING(    0x00, DEF_STR( Off ) )
-	PORT_DIPSETTING(    0x40, DEF_STR( On ) )
-	PORT_DIPNAME( 0x80, 0x00, DEF_STR( Unknown ) )
-	PORT_DIPSETTING(    0x00, DEF_STR( Off ) )
-	PORT_DIPSETTING(    0x80, DEF_STR( On ) )
+	PORT_DIPNAME( 0xc0, 0x00, DEF_STR( Coinage ) )
+	PORT_DIPSETTING(    0x00, "1 Credit per Player" )
+	PORT_DIPSETTING(    0x80, "2 Credits/1 Player, 3 Credits/2 Players" )
+	PORT_DIPSETTING(    0x40, "2 Credits per Player" )
```
Comment (cuavas) [inline src/mame/drivers/galaxian.cpp:5085]: This is just a standard 1C/1P, 3C/2P, 2C/1P setup.  There’s no reason to use custom strings.

File: src/mame/drivers/galaxian.cpp [hunk 2]
```diff
@@ -5079,38 +5079,26 @@ static INPUT_PORTS_START( thepitm )
...
-	PORT_DIPNAME( 0x20, 0x00, DEF_STR( Unknown ) )
-	PORT_DIPSETTING(    0x00, DEF_STR( Off ) )
-	PORT_DIPSETTING(    0x20, DEF_STR( On ) )
-	PORT_DIPNAME( 0x40, 0x00, DEF_STR( Unknown ) )
-	PORT_DIPSETTING(    0x00, DEF_STR( Off ) )
-	PORT_DIPSETTING(    0x40, DEF_STR( On ) )
-	PORT_DIPNAME( 0x80, 0x00, DEF_STR( Unknown ) )
-	PORT_DIPSETTING(    0x00, DEF_STR( Off ) )
-	PORT_DIPSETTING(    0x80, DEF_STR( On ) )
+	PORT_BIT( 0xf0, IP_ACTIVE_HIGH, IPT_UNUSED )
```
Comment (cuavas) [inline src/mame/drivers/galaxian.cpp:5101]: Are these really not physically present?

Comment (scoomby) [reply src/mame/drivers/galaxian.cpp:5101]: > Are these really not physically present?

Yes.  There are just 6 DIP switches as per the photo below:

image

This change brings it in line with the original Moon Quasar board

File: src/mame/drivers/galaxian.cpp (hunk 1 above)
Comment (cuavas) [reply src/mame/drivers/galaxian.cpp:5085]: It’s just showing the actual number of coins though – it’s like the later Midway games in that way.  They typically default to 2 coins per play.  You put in one coin, it shows one coin, and says “insert 1 more coin” or something.  We still call that setting 2 Coins/1 Credit whether the machine displays the raw number of coins or the number of starts you’ve paid for.

Comment (scoomby) [reply src/mame/drivers/galaxian.cpp]: I have implemented the change as advised.  I will make a similar change to the original thepit set for consistency later.  Thanks

**Pull request merged**